packages/styles/scss/components/page-header/_page-header.scss

```diff
@@ -108,10 +130,15 @@
     margin-block-start: $spacing-05;
   }
 
+  [data-hidden]:not([data-fixed]) {
+    display: none;
+  }
+
   .#{$prefix}--page-header__hero-image {
     display: flex;
     overflow: hidden;
     align-items: center;
+    justify-content: end;
```

Comment:
@annawen1  This should be changed to `flex-end` to remove autoprefixer warnings

`(28864:3) autoprefixer: end value has mixed support, consider using flex-end instead`